### PR TITLE
#5519 [Manager] Do not retry the START/STOP APIs for services

### PR DESCRIPTION
**Pull request by eechih**
### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue #5519 
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/it/paper/elements.test.ts [hunk 1]
```diff
@@ -452,34 +452,28 @@ describe('Elements', () => {
         method: 'PUT',
         url: 'api/connectors/*/start**',
         response: {},
-        delay: 1500,
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/paper/elements.test.ts:455]: Thanks :)

**Comment by eechih** [comment]
retry manager-it

**Comment by jackyoh** [comment]
retry manager-it

**Comment by eechih** [comment]
retry manater-e2e

**Comment by eechih** [comment]
retry manager-e2e

**Pull request merged**